### DISCUSSION TITLE
Epoch 9 Reintroduces COFFEE Back Into TEA Project's Ecosystem

### DISCUSSION THREAD

**Discussion by geopor** [tearust/teaproject#122] [Announcements]
Starting in epoch 9, users will purchase COFFEE instead of testnet CML vouchers for our mining contest. 
Read more about [the reintroduction of COFFEE](https://github.com/tearust/teaproject/wiki/Epoch-9) including why we did it and what it means for contestants.

We understand that contestants might have questions about the epoch 9 reintroduction of COFFEE back into the system. Please share your thoughts and feedback in this discussion.

**Comment by Alihlas** [comment]
Good
